CMakeLists.txt

```diff
@@ -82,17 +82,14 @@ option(BUILD_SHELL_INTEGRATION "BUILD_SHELL_INTEGRATION" ON)
 option(BUILD_SHELL_INTEGRATION_ICONS "BUILD_SHELL_INTEGRATION_ICONS" ON)
 
 # this options builds the dolphin integration plugin
-option(BUILD_SHELL_INTEGRATION_DOLPHIN "BUILD_SHELL_INTEGRATION_DOLPHIN" ON)
+option(BUILD_SHELL_INTEGRATION_DOLPHIN "BUILD_SHELL_INTEGRATION_DOLPHIN" OFF)
```

Comment:
This is unrelated to the rest of the pull request. Please revert this change or at least add it as a separate commit. 

Can probably just reverted, as #10770 is about to be worked on which will extract this component anyway.